packages/fast-components-styles-msft/README.md

```diff
@@ -1,2 +1,26 @@
-# FAST components styles - MSFT
-`@microsoft/fast-components-styles-msft` is a collection of JSS (JavaScript Style Sheets) objects and style utilities that power msft components in the FAST ecosystem. The component styles are intended to be compiled with [JSS](https://github.com/cssinjs/jss) and used with base components such as`@microsoft/fast-components-base-react`.
+# FAST Components Microsoft Styles
+A collection of JSS (JavaScript Style Sheets) objects and style utilities that power msft components in the FAST ecosystem. The component styles are intended to be compiled with [JSS](https://github.com/cssinjs/jss) and used with base components such as `@microsoft/fast-components-base-react`, for the styled components see the `@microsoft/fast-components-msft-react` package. The package can also be applied to a custom component.

```

Comment:
If we use `MSFT` in writing we should be consistent everywhere it's written.

Reply:
Resolved